tests/template/template_suite_test.go

```diff
@@ -3,11 +3,10 @@ package template
 import (
 	"testing"
 
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
+	"github.com/openshift/odo/tests/helper"
 )
 
 func TestTemplate(t *testing.T) {
-	RegisterFailHandler(Fail)
-	RunSpecs(t, "TestTemplate Suite")
+	helper.RunTestSpecs(t, "TestTemplate Suite")
```

Comment:
This one doesn't require `RunSpecsWithDefaultAndCustomReporters`, yet calling `RunTestSpecs` will call it, is that alright?

Reply:
Yes, it was a mistake. Someone forgot to update the template when we started using Reproters